src/main/java/com/auth0/json/mgmt/client/Client.java

```diff
@@ -176,19 +178,19 @@ public void setAppType(String appType) {
     }
 
     /**
-     * Getter for the url of the application logo.
+     * Getter for the Uri of the application logo.
      *
-     * @return the application's logo url.
+     * @return the application's logo Uri.
      */
     @JsonProperty("logo_uri")
     public String getLogoUri() {
         return logoUri;
     }
 
     /**
-     * Setter for the application logo url. An image with size 150x150 is recommended.
+     * Setter for the application logo Uri. An image with size 150x150 is recommended.
      *
-     * @param logoUri the logo url to set.
+     * @param logoUri the logo Uri to set.
```

Comment:
url to uri seems okay but why the capitalization?

Reply:
I renamed these to "uri" since that's part of the property's name. And I intended to use caps -> I'll change to URI.